jcl/src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -6,9 +6,14 @@
 
 import com.ibm.oti.vm.VM;
 
+/*[IF Sidecar18-SE-OpenJ9 | Sidecar19-SE]*/
 /*[IF Sidecar19-SE]*/
 import jdk.internal.misc.JavaLangRefAccess;
 import jdk.internal.misc.SharedSecrets;
+/*[ELSE]*/
```

Comment:
This section needs to be commented out in order for this unpre-processed file to compile in tool like Eclipse.

Reply:
fixed